graylog2-web-interface/src/logic/errors/ReportedErrors.js

```diff
@@ -3,18 +3,23 @@
 import { FetchError } from 'logic/rest/FetchProvider';
 
 export const ReactErrorType = 'ReactError';
+export const NotFoundErrorType = 'NotFoundError';
```

Comment:
This constant is not used, from what I can see. I guess we will introduce it later, but let's remove it for now if it's not used yet.